chaoscenter/web/src/controllers/AccountSettingsUserManagement/AccountSettingsUserManagement.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { ExpandingSearchInput } from '@harnessio/uicore';
+import AccountSettingsUserManagementView from '@views/AccountSettingsUserManagement';
+import { useStrings } from '@strings';
+import { useUsersQuery } from '@api/auth/index.ts';
+import { UserType } from '@models';
+
+export default function AccountSettingsUserManagementController(): React.ReactElement {
+  const { getString } = useStrings();
+  const { data, isLoading, refetch: getUsersRefetch } = useUsersQuery({});
+
+  const [includeDisabledUsers, setIncludeDisabledUsers] = React.useState(true);
+  const [searchQuery, setSearchQuery] = React.useState('');
+  const searchInput = (
+    <ExpandingSearchInput placeholder={getString('search')} alwaysExpanded onChange={value => setSearchQuery(value)} />
+  );
+
+  const filteredData = React.useMemo(() => {
+    if (!data) return [];
+    return data.filter(user => {
+      if (user.role === UserType.ADMIN) return false;
+      if (!includeDisabledUsers && user.isRemoved) return false;
+      if (!searchQuery) return true;
+      return (
+        user.name?.toLowerCase().includes(searchQuery.toLowerCase()) ||
+        user.username?.toLowerCase().includes(searchQuery.toLowerCase()) ||
+        user.email?.toLowerCase().includes(searchQuery.toLowerCase())
```

Comment:
filter data in a separate function

Reply:
updated

Reply:
include the entire `{}` after => in the function.

Reply:
updated